src/sentences/aam.rs

```diff
@@ -0,0 +1,130 @@
+use arrayvec::ArrayString;
+use nom::{
+    bytes::complete::is_not,
+    character::complete::{anychar, char},
+    combinator::opt,
+    number::complete::float,
+};
+
+use crate::{parse::NmeaSentence, sentences::utils::array_string, Error, SentenceType};
+
+const MAX_LEN: usize = 64;
+
+/// AAM - Waypoint Arrival Alarm
+///
+/// <https://gpsd.gitlab.io/gpsd/NMEA.html#_aam_waypoint_arrival_alarm>
+///
+/// ```text
+///        1 2 3   4 5    6
+///        | | |   | |    |
+/// $--AAM,A,A,x.x,N,c--c*hh<CR><LF>
+/// ```
```

Comment:
Would you mind adding the explanation of the fields with the number and the example as `code block`?

![Screenshot from 2022-10-26 17-30-14](https://user-images.githubusercontent.com/8925621/198054191-f76083a8-df87-4ca5-a565-d1cc444b3063.png)